src/core/toga/widgets/splitcontainer.py

```diff
@@ -11,7 +11,15 @@ class SplitContainer(Widget):
             If no style is provided then a new one will be created for the widget.
         direction: The direction for the container split,
             either `SplitContainer.HORIZONTAL` or `SplitContainer.VERTICAL`
-        content(``list`` of :class:`toga.Widget`): The list of components to be split.
+        content(``list`` of :class:`toga.Widget`): The list of components to be
+            split or tuples of components to be split and adjusting parameters
+            in the following order:
+        widget (:class:`toga.Widget`): The widget that will be added.
+        weight (float):
+        resize (boolean): Should the content expand when the widget is resized.
+        shrink (boolean): Can the content be made smaller than its requisition.
```

Comment:
I don't think this option needs to exist. If a widget sets a minimum size, that's the widget's minimum size, and a split panel shouldn't be able to override it.